modules/cluster-logging-rn-5.3.14.adoc

```diff
@@ -0,0 +1,84 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-3-14{context}"]
```

Comment:
Usually there is an _ before {context} but I don't think it really matters here. Only change it if you feel like it.